### DISCUSSION TITLE
Sort by genre ?

### DISCUSSION THREAD

**Discussion by Okanda** [DeaDBeeF-Player/deadbeef#2832] [General]
Hello. 
Ican't find a way to sort playlist by genre. Can't find the good format. . . 

thx for your help.

**Comment by Npa972** [comment]
Hi,
Add a column and edit it like this.
![Capture d’écran_2022-06-26_08-15-32](https://user-images.githubusercontent.com/49346431/175813644-a741cc7d-c7f1-45d4-830c-cc854a679a01.png)

**Comment by Okanda** [comment]
Génial. Simple et efficace. 
Merci :)
